prisma/schema.prisma

```diff
@@ -23,7 +23,7 @@ model GDHI {
   itlLevel String  @map("itl_level") @db.VarChar(250)
   itl3     String  @db.VarChar(250)
   region   String  @db.VarChar(250)
-  gdhi2020 Float  @map("gdhi_2020")
+  gdhi Float  @map("gdhi")
```

Comment:
Cheers Daf, fixed on the new branch I pushed (renamed because I realised this branch name was suggesting I was dropping a column rather than renaming it--confused myself!)